index.js

```diff
@@ -25,6 +25,20 @@ module.exports.onRendererWindow = async (window) => {
   if (await isElevated()) {
     window.document.documentElement.classList.add('elevated');
   }
+
+  var config = window.config.getConfig();  
+  var deg = 0;
+
+  const updateBorderAngle = () => {
```

Comment:
Won't doing this in JS be more of a perf issue than letting chromium take care of it?

Reply:
in my system using CSS animations uses double the CPU power than using JavaScript. Sadly.

Reply:
=[. that sucks

Reply:
Yeah. tried it on MacOS and Ubuntu. Furthermore, it is not really possible to animate gradients using CSS animations at the moment, so we had to use hacks that did not look that well.
So unless we find some cool CSS tricks, we should do it with JavaScript.